src/test/scala/chiselTests/IntrinsicSpec.scala

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests
+
+import chiselTests.{ChiselFlatSpec, MatchesAndOmits}
+import circt.stage.ChiselStage
+
+import chisel3._
+
+class IntrinsicSpec extends ChiselFlatSpec with MatchesAndOmits {
+  behavior.of("Intrinsics")
+
+  it should "support a simple intrinsic statement" in {
+    val chirrtl = ChiselStage.emitCHIRRTL(new RawModule {
+      val test = Intrinsic("test")
+    })
+
+    matchesAndOmits(chirrtl)("intrinsic(test)")()
+  }
+
+  it should "support intrinsic statements with arguments" in {
+    val chirrtl = ChiselStage.emitCHIRRTL(new RawModule {
+      val f = 5.U + 3.U
+      val g = f + 2.U
+      val test2 = Intrinsic("test", f, g)
+    })
+
+    matchesAndOmits(chirrtl)("intrinsic(test, f, g)")()
+  }
+  it should "support intrinsic statements with parameters and arguments" in {
+    val chirrtl = ChiselStage.emitCHIRRTL(new RawModule {
+      val f = 5.U + 3.U
+      val g = f + 2.U
+      // TODO: `params = `, implicit conversions, overloads, mixing, oh my!
```

Comment:
As noted in the TODO -- this doesn't work if don't use `params =` here-- thoughts/suggestions welcome, as it's kinda convenient to have the implicit conversions do magic here :).

Reply:
Resolved, split into multiple parameter lists.